pkg/startupmigrations/migrations_test.go

```diff
@@ -90,6 +91,17 @@ type fakeDB struct {
 	putErr  error
 }
 
+// ReadCommittedScan never returns any data.
+func (f *fakeDB) ReadCommittedScan(
+	ctx context.Context, begin, end interface{}, maxRows int64,
+) ([]kv.KeyValue, error) {
+	// Sometimes return the data, sometimes return nothing.
+	if rand.Float64() < .9 {
```

Comment:
I've seen that random number generation is carefully seeded in some other tests sometimes, with `randutil.NewTestRand` or whatnot, does that need to be the case here?